src/components/FeaturedItemsSection/featured-items-section.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { Subtitle, Description, Primary as PrimaryStory, ArgsTable, Stories, PRIMARY_STORY } from '@storybook/addon-docs';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import StackbitFrontmatter from '../../storybook-addons/addon-stackbit-frontmatter/StackbitFrontmatter';
+import StackbitModels from '../../storybook-addons/addon-stackbit-models/StackbitModels';
+import FeaturedItemsSection from './index';
+
+export default {
+    title: 'Components/FeaturedItemsSection',
+    component: FeaturedItemsSection,
+    argTypes: {
+        type: { table: { disable: true } },
+        elementId: {
+            defaultValue: ''
+        },
+        colors: {
+            options: ['colors-a', 'colors-b', 'colors-c', 'colors-d', 'colors-e', 'colors-f', 'colors-g', 'colors-h', 'colors-i'],
+            defaultValue: 'colors-a',
+            control: { type: 'select' }
+        }
+    },
+    parameters: {
+        controls: { expanded: false },
+        docs: {
+            page: () => (
+                <>
+                    <h1 className="normal-case text-3xl">{args.type}</h1>
+                    <Subtitle />
+                    <Description />
+                    <PrimaryStory />
+
+                    <Tabs>
+                        <TabList>
+                            <Tab>Props</Tab>
+                            <Tab>Frontmatter</Tab>
+                            <Tab>Model</Tab>
+                        </TabList>
+
+                        <TabPanel>
+                            <ArgsTable story={PRIMARY_STORY} />
+                        </TabPanel>
+                        <TabPanel>
+                            <StackbitFrontmatter args={args} />
+                        </TabPanel>
+                        <TabPanel>
+                            <StackbitModels args={args} />
+                        </TabPanel>
+                    </Tabs>
+
+                    <Stories />
+                </>
+            )
+        }
+    }
+};
+
+const Template = (args) => <FeaturedItemsSection {...args} />;
+
+const args = {
+    type: 'FeaturedItemsSection',
```

Comment:
args values should match the defaults from the model `models/FeaturedItemsSection.yaml`